.circleci/config.yml

```diff
@@ -471,7 +471,7 @@ jobs:
           time docker pull ${DOCKER_IMAGE}:${DOCKER_TAG} >/dev/null
           export id=$(docker run --env-file "${BASH_ENV}" --cap-add=SYS_PTRACE --security-opt seccomp=unconfined --cap-add=SYS_PTRACE --security-opt seccomp=unconfined -t -d -w /var/lib/jenkins ${DOCKER_IMAGE}:${DOCKER_TAG})
 
-          git submodule sync && git submodule update -q --init --recursive
+          git submodule sync && git submodule update -q --init --recursive --depth 1
```

Comment:
Why do we stop at depth 1 here?

Reply:
Because shallow checkouts are faster (and we don't need commit history during CI builds, do we?)

Reply:
Makes sense. Thanks!